terraform/aks/databases.tf

```diff
@@ -0,0 +1,22 @@
+module "postgres" {
+  source = "./vendor/modules/aks//aks/postgres"
+
+  namespace             = var.namespace
+  environment           = local.app_name_suffix
+  azure_resource_prefix = var.azure_resource_prefix
+  service_short         = var.service_short
+  config_short          = var.config_short
+  service_name          = var.service_name
+
+  cluster_configuration_map = module.cluster_data.configuration_map
+
+  use_azure               = true
```

Comment:
School experience use review apps, and we probably want to create database in containers instead of Azure DBaaS. Can this be changed to a variable such as [deploy_azure_backing_services](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/terraform/aks/variables.tf#L52)?